File: 2026POTD/FEBRUARY/01-02-2026/3010.java
class Solution {

    public int minimumCost(int[] nums) {
        int firstMin = Math.min(nums[1], nums[2]);
        int secondMin = Math.max(nums[1], nums[2]);

        for (int i = 3; i < nums.length; i++) {
            if (nums[i] < firstMin) {
                secondMin = firstMin;
                firstMin = nums[i];
            } else if (nums[i] < secondMin) {
                secondMin = nums[i];
            }
        }

        return nums[0] + firstMin + secondMin;
    }
}

File: 2026POTD/FEBRUARY/01-02-2026/3010.cpp
class Solution {
public:
    int minimumCost(vector<int>& nums) {
        int a=nums[0];
        sort(nums.begin()+1,nums.end());
        int b=nums[1];
        int c=nums[2];
        return a+b+c;       
    }
};